go.mod

```diff
@@ -3,6 +3,7 @@ module go.neonxp.dev/jsonrpc2
 go 1.18
 
 require (
+	github.com/gorilla/websocket v1.5.0
```

Comment:
"The Gorilla Toolkit is now in archive-mode, and is no longer actively maintained. You can read more below." https://github.com/gorilla#gorilla-toolkit
Better to use active project?

Reply:
hi @neonxp  just learned about the gorilla toolkit archiving, sad! 
I think it is best to use an active project, I'm stuck deciding which one to go with since the official golang WebSocket library is also deprecated https://github.com/golang/go/issues/18152 
I would like to know if you have any personal recommendations...